packages/vite/src/node/plugins/importMetaGlob.ts

```diff
@@ -396,10 +396,11 @@ export async function transformGlobImport(
         async ({ globsResolved, isRelative, options, index, start, end }) => {
           const cwd = getCommonBase(globsResolved) ?? root
           const files = (
-            await fg(globsResolved, {
-              cwd,
+            await glob(globsResolved, {
               absolute: true,
+              cwd,
               dot: !!options.exhaustive,
+              expandDirectories: false,
               ignore: options.exhaustive
                 ? []
                 : [join(cwd, '**/node_modules/**')],
```

Comment:
i know this line isn't being changed in the PR but in theory this should do the same in a cleaner way (absolute patterns are automatically converted to relative ones in `tinyglobby`)

```suggestion
                : ['**/node_modules/**'],
```

Reply:
Good point, changed!

Reply:
Seems that it broke some tests, so I reverted.

Reply:
Actually, the failure might be related to catching up with `main` and not the change? Not sure, I'm gonna investigate later.

Reply:
I think the failure was a flaky one. I reran the CI and it passed.